README.md

```diff
@@ -1,7 +1,33 @@
 # Large files storage for eclipse-tractusx.github.io
 
-To publish bigger files through github pages or distribute them, github suggests to use gits large file support: [about-large-files-on-github](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github).
+To publish bigger files through GitHub pages or distribute them, GitHub suggests to use gits large file support: [about-large-files-on-github](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github).
+
+## Naming convention
+
+allowed file names **`DATE_KIT_TITLE_VERSION (e.g., 230808_ECOPASS_VIDEO_v1)`**
+
+- DATE: YYMMDD
```

Comment:
Do you think using `DATE` in the name is a good idea? What is exactly is the date: date of publishing? Date of upload? Data last changed?

I would prefer to use the current version based on CalVer, e.g. 23.12.

Reply:
IMHO i would prefer date of publishing, and some information / videos presentation will stay i guess longer than suggested CalVer. 😉

Reply:
Hmm. CalVer would assume, that resources are always published for a specific Tractus-X release and I'm not sure, if that's always the case.
The Date on the other hand also does not really give any additional info, that one would get out of the git history.
It would only sort the files by "added". If we omit the date, the files would be sorted by component, which would be easier to search in my opinion

Reply:
Thanks for your input so far, 

guess than we need a little vote here

Reply:
I like the idea for CalVer (please vote with a +1 👍🏼 ) for this comment

Reply:
I like the idea for publishing date (please vote with a +1 👍🏼 ) for this comment

Reply:
What do you think about just removing the date without CalVer or anything else? So something like `KIT_TITLE_VERSION` as a name, since the date is always included in the file details or git history.

Reply:
remove DATE completly from naming convention because of higlighted information from @SebastianBezold under aggreement from @mhellmeier  with [924e949](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io.largefiles/pull/7/commits/924e949983d3db6c571901a5337b79ced3f6dc5b)

>The Date on the other hand also does not really give any additional info, that one would get out of the git history.

Reply:
Great, thank you!